crates/ruma-events/src/room/member.rs

```diff
@@ -92,6 +92,13 @@ pub struct RoomMemberEventContent {
     #[cfg(feature = "unstable-pre-spec")]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub reason: Option<String>,
+
+    /// Arbitrarily chosen `UserId` (MxID) of a local user who can send an invite.
+    #[cfg(feature = "unstable-pre-spec")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "unstable-pre-spec")))]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[ruma_event(skip_redaction)]
+    pub join_authorised_via_users_server: Option<UserId>,
```

Comment:
Oh no, did the spec introduce a British English name once again? If this is really what the spec says, can you add a `serde(rename)` but use `authorized` (with `z`) for the field name, for consistency within Ruma?

Reply:
There's 25 matches for `authoriz` (and 11 for `authoris`) on https://spec.matrix.org/v1.1/client-server-api/, so if there is the policy at all, they are not very good at enforcing it 😅